### PR TITLE
user profile translations

### LINKED ISSUE #383: Improvements Post merge of Login branch
- [x] Add translations in all the files. 
- [x] Translate months
- [x] Translate the date pickers
- [x] Signup Validations
- [x] Email Verification
- [x] Add multiple choice picker for owner type
- [x] User should not be able to add 2 reports for the same year.

In Project Sites 

- [x] Design on Safari
- [x] Delete should be replaced by Clear all
- [x] Another option to delete single
- [x] There should be add polygon button instead of polygon button
- [x] Disable drawing by default

### PR DESCRIPTION
**Pull request by captainamiedi**
Fixes #383 

Changes in this pull request:
- translate user details
-
-

@sagararyal

### CODE REVIEW

File: public/static/locales/en/me.json [hunk 1]
```diff
@@ -5,5 +5,10 @@
   "registerTrees": "Register Trees",
   "redeem": "Redeem",
   "myForest": "My Forest",
-  "giftToGiftee": "Gift to ${gifteeName}"
+  "giftToGiftee": "Gift to ${gifteeName}",
+  "settingManageProject": "Manage Projects",
+  "userTree": "Register Trees",
+  "userBio": "{{userBioName}}",
+  "logout": "Logout",
+  "displayName": "{{name}}"
```
Comment (sagararyal) [inline public/static/locales/en/me.json:13]: why is "userBio" and displayName this here?

File: src/features/public/Donations/components/SingleProjectDetails.tsx [hunk 2]
```diff
@@ -240,7 +244,9 @@ function SingleProjectDetails({ project }: Props): ReactElement {
                 ideal={350}
                 max={400}
                 readMoreText={t('donate:readMore')}
-                text={project.description}
+                text={t('donate:readMoreHalfText', {
```
Comment (sagararyal) [inline src/features/public/Donations/components/SingleProjectDetails.tsx:247]: why is this here as well? project description comes from API

File: public/static/locales/en/forest.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "myForest": "MY FOREST",
+    "forestName": "{{name}}",
+    "forestCountry": "{{country}}",
+    "forestGift": "{{gift}}"
```
Comment (sagararyal) [inline public/static/locales/en/forest.json:5]: This looks useless as well

**Review by sagararyal — CHANGES_REQUESTED** [review]
Please remove unnecessary strings

**Review by norbertschuler — APPROVED** [review]
I have not checked if there are any text strings missed, but the patch looks ok for me.

**Comment by sagararyal** [comment]
@norbertschuler 
The Project Name is not displayed on the List.
Screen Shot 2020-11-09 at 9 24 04 PM

File: src/features/user/UserProfile/components/MyForestContainer.tsx [hunk 4]
```diff
@@ -13,14 +15,15 @@ export default function MyForestContainer({ userprofile }: any) {
...
+  const {t} = useTranslation(['forest']);
 
   const userForests = userprofile.projects.map((forest: any) => {
     return <MyForestItem key={forest.id} forest={forest.name} />;
   });
   return (
     // the text and two boxes
     <div className={styles.outerMyForestContainer}>
-      <h6 className={styles.myForestTitleText}> MY FOREST </h6>
+      <h6 className={styles.myForestTitleText}> {t('forest:myForest')} </h6>
```
Comment (norbertschuler) [inline src/features/user/UserProfile/components/MyForestContainer.tsx]: A forest.json file does not exists yet!

File: src/features/user/UserProfile/components/UserInfo.tsx [hunk 5]
```diff
@@ -2,13 +2,17 @@ import React from 'react';
...
 
+const {useTranslation} = i18next;
 export default function UserInfo({
   userprofile,
   handleTextCopiedSnackbarOpen,
   authenticatedType,
   handleAddTargetModalOpen
 }: any) {
+  const {t} = useTranslation(['me']);
+  console.log(userprofile);
```
Comment (norbertschuler) [inline src/features/user/UserProfile/components/UserInfo.tsx]: Remove console.log before merging this patch.

**Review by norbertschuler — CHANGES_REQUESTED** [review]

**Review by norbertschuler — APPROVED** [review]

**Comment by norbertschuler** [comment]
> @norbertschuler
> The Project Name is not displayed on the List.

@sagararyal I fixed a lot of bugs introduced with this patch. Hopefully it is correct by now.

**Pull request merged**